views/credits.ejs

```diff
@@ -0,0 +1,57 @@
+<!--
+Copyright © 2017, ACM@UIUC
+
+This file is part of the Groot Project.  
+
+The Groot Project is open source software, released under the University of
+Illinois/NCSA Open Source License.  You should have received a copy of
+this license in a file with the distribution.
+-->
+
+<%- include('_partials/header') -%>
+
+<div id="credits-container">
+  <div class="row col-centered small-12 medium-6 large-6">
+    <%- include('_partials/flash') -%>
+    <h3>Credits</h3>
+    <div>
+      <h5 style="display:inline; float:left; padding-top:20px" class="balance">Current Balance: <b class="balance_amount">$<%= Number(balance).toFixed(2) %></b></h5>
+      <a class="button" name="refill" href="/credits/addFunds" style="float:right">Refill Balance</a>
+    </div>
+    <% if (transactions && transactions.length != 0) { %>
+      <table id="transactions-table" class="dynatable-container">
+        <thead>
+          <tr>
+            <th> Id </th>
+            <th> Amount </th>
+            <th> Time </th>
+            <th> Description </th>
+          </tr>
+        </thead>
+        <% for (var t of transactions) { %>
+          <tr class="transaction">
+            <td> <%= t.id %> </td>
+            <td> <%= (t.amount >= 0 ? '$' : '-$') + Math.abs(t.amount).toFixed(2) %> </td>
+            <td> <%= t.created_at %> </td>
+            <td> <%= t.description ? t.description : "No description" %> </td>
+          </tr>
+          <% } %>
+        </table>
+    <% } else { %>
+      <h5>There are no transactions.</h5>
+    <% } %>
+  </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
Another way to do this is just to load this below the footer, I think, since footer loads all javascript. Unless its better practice to add an event listener onto the window (in which case, it's fine.)